TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/autonomous/CraterStartOpMode.java

```diff
@@ -0,0 +1,40 @@
+package org.firstinspires.ftc.teamcode.opmodes.autonomous;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.teamcode.systems.drive.DriveSystem4Wheel;
+
+@Autonomous(name = "CraterStartOpMode", group = "Bot")
+public class CraterStartOpMode extends BaseAutonomousOpMode {
+    private final String TAG = "CraterStartOpMode";
+
+    public CraterStartOpMode () {
+        super("CraterStartOpMode");
```

Comment:
TAG?

Reply:
I could be wrong here, and we might decide we don't like this, but in Android I've learned as a convention to put a TAG string with the debugging tag for that class, then when logging call telemetry.log(TAG, "whatever message you want to put");. That way you know what class the telemetry is coming from in the logs.